assert_release_quality

```diff
@@ -1,8 +1,20 @@
 #!/bin/sh
+#
+# Asserts that the module is ready for release with the expected quality
 
 # cf. https://docs.npmjs.com/cli/version
 new_version=$1
 
+# Checking that the npm-shrinkwrap.json file is up-to-date wrt to the
+# package.json file.
+if [ package.json -nt npm-shrinkwrap.json ]
+then
+    echo "File package.json is newer than file npm-shrinkwrap.json"
```

Comment:
Yeah, I can't think of any good solutions. I usually just manually see if the dependencies changed in the package.json file and verify that the shrinkwrap file was also updated. But this may not be a big deal here since this is just in a `assert_release_quality.sh` script and not some Git commit hook that would get run by contributors.